### DISCUSSION TITLE
Are your minio healthchecks still working?

### DISCUSSION THREAD

**Discussion by ovizii** [minio/minio#18375] [Q&A] [answered]
Mine seems broken since the latest update. I entered the container with `docker exec -ti minio-server bash` intending to manually try the healthcheck with `curl -f http://localhost:9000/minio/health/live` and there is apparently no curl available. 

Can anyone confirm?

**Comment by klauspost** [comment, marked as answer]
See https://github.com/minio/minio/issues/18373

**Reply by ovizii** [reply]
Thanks a million, my search only yielded very old threads from about the time when this healthscheck was being implemented :-) 

Also, the documentation seems to still mention curl: https://min.io/docs/minio/linux/operations/monitoring/healthcheck-probe.html

**Reply by harshavardhana** [reply]
curl is mentioned for outside entities see the DNS name it uses - it is never meant as part of the container that is shipped.

Now if you insist on using Docker's healthchecks embrace `mc ready local`
